Lib/urllib/robotparser.py

```diff
@@ -56,10 +56,10 @@ def set_url(self, url):
         self.url = url
         self.host, self.path = urllib.parse.urlparse(url)[1:3]
 
-    def read(self):
+    def read(self, sslcontext=None):
```

Comment:
I'd prefer to add this to the constructor of RobotFileParser instead of the read() method.